projects/igniteui-angular/src/lib/grid/column-hiding.component.ts

```diff
@@ -78,24 +125,44 @@ export class IgxColumnHidingComponent extends ColumnChooserBase {
         }
         return item;
     }
-
+    /**
+     * Shows all columns in the grid.
+     * ```typescript
+     * this.columnHiding.showAllColumns();
+     * ```
+     * @memberof IgxColumnHidingComponent
+     */
     public showAllColumns() {
         for (const col of this.hidableColumns) {
             col.value = false;
         }
     }
-
+    /**
+     * Hides all columns in the grid.
+     * ```typescript
+     * this.columnHiding.hideAllColumns();
+     * ```
+     * @memberof IgxColumnHidingComponent
+     */
     public hideAllColumns() {
         for (const col of this.hidableColumns) {
             col.value = true;
         }
     }
-
+    /**
+     * Emits an event when а column visibility has changed.
+     * ```typescript
+     * this.columnHiding.onVisibilityChanged({column:firstColumn, newValue:true});
+     * ```
+     * @memberof IgxColumnHidingComponent
+     */
```

Comment:
Actually hide this onVisibilityChanged method, it is used for internal implementacion of the same named event